packages/financial-templates-lib/src/clients/InsuredBridgeL1Client.ts

```diff
@@ -190,6 +188,11 @@ export class InsuredBridgeL1Client {
     return toBN(await this.bridgeAdmin.methods.proposerBondPct().call());
   }
 
+  async getL2DepositBoxAddress(chainId: number): Promise<string> {
```

Comment:
this method was in the l2 client. however, it should be here in the l1 client as it is done against l1 contracts. more over, we already have the l1 web3 stance and the BridgeAdmin so no passing in required. I made associated changes to the `index` for the relayer and the `CreatePriceFeed` logic that called these methods.